pkgs/by-name/ne/neocities-cli/package.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, bundlerApp
+, bundlerUpdateScript
+, installShellFiles
+}:
+
+let
+  ttBundlerApp = bundlerApp rec {
+    pname = "neocities";
+    gemdir = ./.;
+    exes = [ pname ];
```

Comment:
more on pname

https://github.com/NixOS/nixpkgs/issues/277994

Reply:
so should I just hardcode `exes` attribute ?

Reply:
yes

have a look at these

```
[kirillvr@tsutenkaku:~/nixpkgs/pkgs/by-name]$ rg bundlerApp
pd/pdk/package.nix
ma/maid/package.nix
ba/bashly/package.nix
pr/pru/package.nix
yo/youplot/package.nix
```

Reply:
I've done that in the last commit, let me know if it's right